.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,164 @@
+---
+name: 🐛 Bug Report
+description: |
+  Submit a bug report to help us improve
+#title: "[Bug]"
+labels: ["bug", "P2-medium"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## ❗ Important Announcement
+
+        ### 🚫 Please Avoid Unnecessary Pinging of Maintainers
+
+        **We kindly ask users to refrain from pinging maintainers unless absolutely necessary. Pings should be reserved for critical issues or urgent matters that require immediate attention.**
+
+        - **Why**: To help maintainers focus on high-priority tasks, we kindly request that unnecessary or repeated pings be minimized. This will help ensure quicker responses for matters that truly need attention.
+        - **What This Means**: Non-urgent pings may be ignored or addressed later. Please assess the urgency of your request before pinging a maintainer.
+
+        #### Guidelines for Pinging
+
+        - ✅ **Appropriate**: Critical issues or urgent matters requiring immediate attention.
+        - ❌ **Not Appropriate**: Repeated or non-urgent pings, which may be ignored or addressed later.
+
```

Currently, this is quite wordy (the form is quite long => too long forms don't get read).. lets rephrase it as such

```suggestion
        🚫 **We kindly ask you to refrain from pinging maintainers unless absolutely necessary. Pings are reserved for critical/urgent issues that require immediate attention.**

        *Why*: Reserving pings for urgent matters ensures maintainers can prioritize critical tasks effectively
```